src/modules/pos/validator/mocks/index.js

```diff
@@ -95,3 +96,20 @@ export const generators = rest.get(`*/api/${API_VERSION}/generators`, async (req
 export const posConstants = rest.get(`*/api/${API_VERSION}/dpos/constants`, async (_, res, ctx) =>
   res(ctx.delay(20), ctx.json(mockPosConstants))
 );
+
+export const rewardsLocked = rest.get(`*/api/${API_VERSION}/pos/rewards/locked`, async (req, res, ctx) => {
```

Comment:
```suggestion
export const posRewardsLocked = rest.get(`*/api/${API_VERSION}/pos/rewards/locked`, async (req, res, ctx) => {
```

Reply:
Should i prefix everything with pos as well? usePosRewardsLocked instead of useRewardsLocked

Reply:
no just the api call function is the same name as edndpoint